### ISSUE TITLE
[1.12.2] /ct syntax doesn't print errors in chat in 4.1.20.620

### ISSUE THREAD

**Issue by TCreopargh** [CraftTweaker/CraftTweaker#1179]
#### Issue Description:  

in 4.1.20.620 /ct syntax only prints obvious syntax errors like spamming a lot of meaningless brackets, but nothing else will be printed in chat such as non-existent types, functions etc
In 4.1.20.619 it prints all errors in the chat.
Also no error message is sent to the chat when joining a world, this isn't the case in 4.1.20.619

#### What happens:  
/ct syntax says that this script is correct:
```ZenScript
fsdfsg.dngidnsfigisdfgni() = fdsfsddgsg;
var sdffsdgfsg as dftsg = sdffjidsfjisdjgij;
```

#### What you expected to happen:  
It should print errors in the chat, which is the case prior to version 619
Or, if this change is intended, it should at least say that there are non-syntax errors when doing /ct syntax. When there are a lot of scripts it's very hard to find errors in the log.

#### Script used:  

Codeblock above

#### crafttweaker.log file:  

It prints error in the log when loading the game and prints errors in the log when doing /ct syntax, but printing them in the chat is obviously better for debugging
https://paste.ubuntu.com/p/JzXSsQCGNV/
____
#### Environment:  

- Minecraft Version: 1.12.2
- Forge Version: 14.28.5.2854 
- CraftTweaker Version: 4.1.20.620
- Are you using a server: no
- If yes, does the client have the exact same scripts?

**Comment by democat3457** [comment]
does /ct syntax work if you're on a server and opped?

**Comment by TCreopargh** [comment]
> does /ct syntax work if you're on a server and opped?

Nope, still doesn't work as intended

**Comment by jaredlll08** [comment]
Just pushed an attempted fix, please try it out.

note that I don't even bother trying to load 1.12 anymore, so while my fix should be the *correct* way to see if a player is an OP (which is used to determine if they should get the message or not), it may not be correct for this usecase.

**Comment by TCreopargh** [comment]
Nope, Still doesn't work in version 621

**Comment by TCreopargh** [comment]
Fixed in 4.1.20.622
